eth/tracers/tracers_test.go

```diff
@@ -124,7 +206,7 @@ func TestCallTracer(t *testing.T) {
 		t.Fatalf("failed to retrieve tracer test suite: %v", err)
 	}
 	for _, file := range files {
-		if !strings.HasPrefix(file.Name(), "call_tracer_") {
+		if !strings.HasPrefix(file.Name(), "call_tracer_create2") {
```

Comment:
Why should we add this change?

Reply:
Oh that's a mistake. I experimented with a separate json testcase, but later abandoned that, and made a totally separate test. So that's just leftover crap...